From: Ahmad Fatoum <a.fatoum@pengutronix.de>
Date: Tue, 15 Feb 2022 14:20:55 +0100
Subject: [PATCH] common: add help text for CONFIG_STATE

With the addition of barebox TLV, a help text differentiating it from
barebox-state is in order.

Signed-off-by: Ahmad Fatoum <a.fatoum@pengutronix.de>
---
 common/Kconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/common/Kconfig b/common/Kconfig
index f7a6a96e877c..3ff45d6c9d0d 100644
--- a/common/Kconfig
+++ b/common/Kconfig
@@ -1004,6 +1004,11 @@ config STATE
 	select ENVIRONMENT_VARIABLES
 	select OFTREE
 	select PARAMETER
+	help
+	  barebox state is a generic framework for atomic power fail-safe
+	  variable storage and retrieval. It can be used to safely maintain
+	  data over reboots and to exchange information with Linux, e.g.
+	  for redundant boot with bootchooser.
 
 config STATE_CRYPTO
 	bool "HMAC based authentication support"
